### PR TITLE
Add PR template

### PR DESCRIPTION
**Pull request by aldenpeterson-wf**
@Workiva/messaging-pp @brettkail-wk

### CODE REVIEW

**Comment by aviary-wf** [comment]
## Raven
<details>
<summary>Number of Findings: 0</summary>

</details>

**Comment by brianshannan-wf** [comment]
Did we ever get our github group configured so people not in it could @ mention it?

**Comment by aldenpeterson-wf** [comment]
I'm not sure that's possible based on how GH groups work, though I'm not 100% sure.

File: .github/PULL_REQUEST_TEMPLATE.md [hunk 1]
```diff
@@ -0,0 +1 @@
+Pull requests should be made against the 'develop' branch, not master.
```
Comment (brettkail-wk) [inline .github/PULL_REQUEST_TEMPLATE.md]: Ha, clever :-).

**Comment by jacobmoss-wf** [comment]
@brianshannan-wf Yes, it did.

@aldenpeterson-wf this is already in https://github.com/Workiva/frugal/blob/master/CONTRIBUTING.md

**Comment by aldenpeterson-wf** [comment]
@jacobmoss-wf this template populates a PR with the text when you open the PR to make it more obvious.

**Comment by brianshannan-wf** [comment]
+1

**Comment by aldenpeterson-wf** [comment]
I don't know how to verify this, either. I guess it really should be against master, too, ironically so that it goes live before the next release.

**Comment by stevenosborne-wf** [comment]
+1

**Pull request merged**